app/views/tasks/_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= simple_form_for [@task] do |t| %>
+    <%= t.error_notification %>
+    <%= t.error_notification message: t.object.errors[:base].to_sentence if t.object.errors[:base].present? %>
+
+    <div class="form-inputs">
+      <%= t.input :description %>
+      <%= t.input :priority %>
+      <%= t.input :contact_id %>
+      <%= t.input :date_time %>
+      <%= t.input :ownership %>
+      <%= t.input :complete %>
+      <%= t.input :category %>
```

Comment:
this is a good start! next step is A) do we need all of these fields? which one will have default values that we set? and B) which fields would be better to be a different kind of input (e.g. dropdown / date picker, etc)